All EIPIP Meetings/Meeting 060.md

```diff
@@ -0,0 +1,121 @@
+
+**Micah:** a similar note that is license for a reference implementation discussion so um on this 157 issue there's two separate things one is should we require everything to be a certain license or not and two should we lint people's code i'm pretty against linking people's code because i want authors to have autonomy to write their code how they want and if you want to do curly braces on next line with you know vertical alignment and using spaces then that's your choice and i don't want to try to enforce on anyone else any particular um this is mainly because i don't want editors to be taking philosophical positions on things at all possible so that's the one thing i'm fairly
+
+**Pooja:** against that one the licensing one is more separate more complicated we do need to resolve that because right now we don't have consistency and we should decide you know what licenses are people allowed to include as part of their eip i'm pretty strongly in favor of everything should be cc0 like the ipsc 0 and that means everything the eip you can't get out of this cc0 license by just moving all your content over to an asset file like i think that's not an appropriate way to bypass the cc0 licensing but i know not everybody agrees with me on this and so i do think we need more discussion yeah i think the other side of it is i think we should allow other licenses in the assets directory like cc0 isn't even an osi approved license so yeah it's because osi is dumb they have a pretty good reason for it i think um it explicitly does not grant patents where you know the unlicensed or some of the other cc0s licenses do yeah sure but neither does mit and i believe mit is no no exactly yes mit doesn't mention patents at all whereas cc0 explicitly does not grant patents and they were concerned about that why would we want people have patents over eips we don't know it means that you can get pen the fear is patent trolls so someone comes in licensed something with cc0 but it includes a patent infringing code and then someone else will then use that code because it stacy zero license and then the patent 
```

Comment:
```suggestion
**Micah:** so that's the one thing i'm fairly against that one the licensing one is more separate more complicated we do need to resolve that because right now we don't have consistency and we should decide you know what licenses are people allowed to include as part of their eip. i'm pretty strongly in favor of everything should be cc0 like the eip is cc0 and that means everything the eip you can't get out of this cc0 license by just moving all your content over to an asset file like i think that's not an appropriate way to bypass the cc0 licensing but i know not everybody agrees with me on this and so i do think **we need more discussion**.

 **Sam**: yeah i think the other side of it is i think we should allow other licenses in the assets directory like cc0 isn't even an osi approved license so 
 
 **Micah**: yeah it's because osi is dumb
 
**Sam**:  they have a pretty good reason for it i think um it explicitly does not grant patents where you know the unlicensed or some of the other cc0s licenses do 

**Micah**: yeah sure but neither does mit and i believe mit is 

**Sam**: no no exactly yes mit doesn't mention patents at all whereas cc0 explicitly does not grant patents and they were concerned about that 

**Gavin**: why would we want people have patents over eips 

**Sam**: we don't 

**Micah**: the fear is patent trolls. so someone comes in licensed something with cc0 but it includes a patent infringing code and then someone else will then use that code because it CC0 license 
```